deploy/crds/crd-issuers.yaml

```diff
@@ -1886,9 +1881,14 @@ spec:
                                                 name:
                                                   description: |-
                                                     Name of the referent.
-                                                    More info: https://kubernetes.io/docs/concepts/overview/working-with-objects/names/#names
+                                                    This field is effectively required, but due to backwards compatibility is
+                                                    allowed to be empty. Instances of this type with an empty value here are
+                                                    almost certainly wrong.
                                                     TODO: Add other useful fields. apiVersion, kind, uid?
+                                                    More info: https://kubernetes.io/docs/concepts/overview/working-with-objects/names/#names
+                                                    TODO: Drop `kubebuilder:default` when controller-gen doesn't need it https://github.com/kubernetes-sigs/kubebuilder/issues/3896.
```

Comment:
comment: Not anything that needs to change in this PR, but it's annoying to see "TODOs" in our CRD docs like this - shame that the generator isn't stripping them out